command-not-found.nu

```diff
@@ -0,0 +1,40 @@
+{ |cmd_name|
+  let install = { |pkgs|
+    $pkgs | each {|pkg| $"  nix shell nixpkgs#($pkg)" }
+  }
+  let run_once = { |pkgs|
+    $pkgs | each {|pkg| $"  nix shell nixpkgs#($pkg) --command '($cmd_name) ...'" }
+  }
+  let single_pkg = { |pkg|
+    let lines = [
+      $"The program '($cmd_name)' is currently not installed."
+      ""
```

Comment:
not sure this would be worth taking into consideration, but the (minor) formatting differences here could have some unintended effects on command-parsing tools (think [thefuck](https://github.com/nvbn/thefuck), [pay-respects](https://codeberg.org/iff/pay-respects))

Reply:
Interesting. I never used such tools so I didn't take that into consideration. Shall I install those tools locally and compare their behavior when using `command-not-found.sh` vs `command-not-found.nu`?

Reply:
The `manifest.json` check seems to tell us whether the user is using Nix profiles or not. And not whether they enabled flakes or not. However, I never used profiles / `manifest.json` so I'm kinda lost here. So we have four cases now:

1. Flakes disabled / profiles disabled
2. Flakes enabled / profiles disabled
3. Flakes disabled / profiles enabled
4. Flakes enabled / profiles enabled